### ISSUE TITLE
Reconstruction completed - Cannot find dense model .PLY file

### ISSUE THREAD

**Issue by Parattata** [colmap/colmap#1006]
I've installed COLMAP on my Windows 10 OS. Installation goes smoothly and the program seems to be just fine.

I downloaded the Gerrard Hall image dataset to experiment. The automatic reconstruction, with sparse and dense models enabled, took around 2,5 hours to complete. This has created some folders in the project directory, the dense folder being one of them. I cannot seem to find any .ply file to view the dense model though! The program viewport shows the sparse model by default, after the reconstruction completion.

Any suggestions?

**Comment by tsattler** [comment]
I never used the automatic reconstruction pipeline, but if it runs the point fusion step, there should be a .ply file in the dense folder.

**Comment by Parattata** [comment]
> 
> 
> I never used the automatic reconstruction pipeline, but if it runs the point fusion step, there should be a .ply file in the dense folder.

In order to avoid a long re-reconstruction process, do you think that the Point Fusion step would be recorded in the log file? So that I can just check it instantly. I looked up for "point fusion" and "fusion", but the word search returned no results.

**Comment by tsattler** [comment]
I would assume so.

Did you had a look at this: https://colmap.github.io/tutorial.html#quickstart ?

**Comment by Parattata** [comment]
> 
> 
> Did you had a look at this: https://colmap.github.io/tutorial.html#quickstart ?

Of course. I'm aware of the tutorial, that's how I got to know that I'm missing the .ply file from the dense folder in the first place.

By the way, I'm dropping the log file here, should anyone wish to review it.
[gerard hall log 20201013001.log](https://github.com/colmap/colmap/files/5376902/gerard.hall.log.20201013001.log)

**Comment by tsattler** [comment]
Looking at your log file, it seems that depth maps were not computed since CUDA is required for this task and CUDA could not be found. Without the depth maps, it is not possible to construct a dense model, which is the reason why there is no .ply file in the directory.

**Comment by Parattata** [comment]
Thank you very much! I see now. NVidia beats the competition here, sadly. Is CUDA a universal requirement for all photogrammetry software out there? If so, that's a serious barrier.

**Comment by tsattler** [comment]
There should be CPU only methods as well. But CUDA is often used for dense reconstruction for efficiency.
